EnglishFilter/sections/general_extensions.txt

```diff
@@ -235,6 +235,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79481
+hocvientruyentranh.net#%#//scriptlet("abort-on-property-read", "popup_link")
```

Comment:
I can't reproduce popups and I can't find popup_link variable/function in site's code

Reply:
@wout-codes 
Please check do we still need this rule?

Reply:
Yes, I can still reproduce sometimes, not everytime.

Reply:
@AdguardTeam/filters-maintainers please take a look

Reply:
I still can't reproduce popups and I can't find popup_link variable/function in site's code
@wout-codes could you please make screenshot with popup_link in code?

Reply:
@Sergey-Lyapin, popup not reproduced on my side either.